qa/workunits/windows/test_rbd_wnbd.py

```diff
@@ -234,78 +283,58 @@ def create(cls,
     def get_disk_number(self,
                         timeout: int = 60,
                         retry_interval: int = 2):
-        tstart: float = time.time()
-        elapsed: float = 0
-        LOG.info("Retrieving disk number: %s", self.name)
-        while elapsed < timeout or not timeout:
+
```

Comment:
Nit: unneeded whitespace

Reply:
the pep8 linter usually requires this, double checking

Reply:
That would mean that there is required whitespace missing in `_wait_for_disk` then.